core/src/com/unciv/ui/tutorials/TutorialController.kt

```diff
@@ -0,0 +1,49 @@
+package com.unciv.ui.tutorials
+
+import com.unciv.UncivGame
+import com.unciv.models.Tutorial
+
+interface TutorialController {
```

Comment:
Why would you create an interface with just one class which is an implementation, instead of just having it as a concrete class? What's the benefit of the interface?